index.d.ts

```diff
@@ -0,0 +1,32 @@
+import Vue, { ComponentOptions } from "vue";
+
+declare global {
+  namespace Cypress {
+    import Vue from "vue";
```

Comment:
why the second `import` of Vue?

Reply:
Resolved. Not sure this did anything, but it works just fine without it.